src/test/resources/projects/it1/reference/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Sonatype Nexus (TM) Open Source Version
+    Copyright (c) 2018-present Sonatype, Inc.
```

Comment:
A note: all of these years are wrong (it should be 2019). That's likely just something we need to fix in the archetype and unrelated to this, but worth pointing out.

Reply:
Cool. I will use this as a chance to intentionally break the IT suite, and verify CI. :)